lib/routes/account.js

```diff
@@ -960,6 +976,9 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
               // `email-captcha` is a custom verification method used only for
               // unblock codes. We do not need to send a verification email
               // in this case.
+            } else if (verificationMethod === 'totp-2fa') {
```

Comment:
Should we refactor this whole chain of tests to a single `switch`? Repeatedly testing the same string seems a bit verbose/expensive.